### PR TITLE
debian: set postgres db parameter

### PR DESCRIPTION
**Pull request by joergsteffens**
The Bareos catalog should be created with following settings:
Encoding: SQL_ASCII
Collate:  C
Ctype: C
While all recent versions of dbconfig support the parameter Encoding,
Collate and Ctype are only supported in dbconfig >= 2.0.21.
On older versions, these setting is ignored.

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

##### General

- [x] PR name is meaningful
- [x] Purpose of the PR is understood
- [x] Separate commit for this PR in the CHANGELOG.md, PR number referenced is same
- [x] Commit descriptions are understandable and well formatted

##### Source code quality

- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- ~Required documentation changes are present and part of the PR~
- [ ] `bareos-check-sources --since-merge` does not report any problems
- [x] `git status` should not report modifications in the source tree after building and testing

### CODE REVIEW

File: core/README.dbconfig
Comment (bruno-at-bareos) [inline core/README.dbconfig:89]: Shouldn't we already remove all mysql / sqlite ?

Comment (joergsteffens) [reply core/README.dbconfig:89]: I decided against it, as this PR is not about removing MySQL/Sqlite.

**Review by bruno-at-bareos — APPROVED** [review]
Ready.

**Comment by arogge** [comment]
Please don't merge before 1032, merging the CHANGELOG.md would be even worse bad otherwise

**Review by arogge — APPROVED** [review]

**Pull request merged**
